src/firefly/js/charts/chartTypes/HistogramTblView.jsx

```diff
@@ -38,7 +40,9 @@ function Chart(props) {
     if (!TblUtil.isFullyLoaded(tblId) || !chartData || !heightPx || !widthPx) {
         return (<div/>);
     }
-    const { isDataReady, data:histogramData, options:histogramParams} = ChartsCntlr.getChartDataElement(chartId);
+    var { isDataReady, data:histogramData, options:histogramParams} = ChartsCntlr.getChartDataElement(chartId);
+    const HistogramInstance = get(getAppOptions(), 'charts.chartEngine') !== 'plotly' ? Histogram : HistogramPlotly;
```

Comment:
I suggest renaming Histogram to HistogramHighcharts and have Histogram.jsx handle instance creation, like you do here. This way all references to Histogram will be switched to plotly references without changes in the code that uses it. (We have a few references besides HistogramTblView.)